ambry-api/src/main/java/com.github.ambry/store/Message.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2018 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.store;
+
+import java.io.InputStream;
+
+
+/**
+ * Representation of a message in the store. Contains the {@link MessageInfo} and the {@link InputStream} of data.
```

Comment:
The javadoc looks out of date since the member is a byte array rather than InputStream.

Reply:
Thanks - this is being re-written now to use an `InputStream`, but I'll make sure that the doc is in sync.